### DISCUSSION TITLE
CoD4x:: No autoupdate!

### DISCUSSION THREAD

**Discussion by Buenni** [GameServerManagers/LinuxGSM#3808] [Q&A] [answered]
Hello, everyone,
I installed a CoD4x server with LinuxGSM today and I have the problem that the server does not automatically update to the lastest version. I have already tried it manually, only then the CoD4x server does not start anymore!

I hope someone here is able to help me?!

Thank you in advance!

Best regards, Buenni

**Comment by Buenni** [comment]
I solved the problem myself!

Download the cod4x_dedrun to your server with wget, but first delete the old one!

**Comment by h3o66** [comment, marked as answer]
Fyi: this should be also solved with the latest lgsm release 22.2.0
